connection.go

```diff
@@ -1724,6 +1743,7 @@ func (s *connection) checkTransportParameters(params *wire.TransportParameters)
 
 func (s *connection) applyTransportParameters() {
 	params := s.peerParams
+	fmt.Println("apply transport parameters", s.config.MaxIdleTimeout, params.MaxIdleTimeout)
```

Comment:
Leftover debug statement. Merged via this PR, then fix using force-push on master. Sorry for that.